core/src/avm2/globals.rs

```diff
@@ -90,6 +90,7 @@ pub struct SystemClasses<'gc> {
     pub sharedobject: ClassObject<'gc>,
     pub mouseevent: ClassObject<'gc>,
     pub progressevent: ClassObject<'gc>,
+    pub gestureevent: ClassObject<'gc>,
```

Comment:
This can be removed, as this field isn't used anywhere

Reply:
Thanks!